app/views/users/show.html.erb

```diff
@@ -1 +1,23 @@
-<h1>This is going to be the show page</h1>
\ No newline at end of file
+<section class="my-bio bg-secondary bg-gradient">
+  <div class="container">
+    <div class="row">
+      <%= render partial: 'user', locals: { user: @user } %>
+    </div>
+    <div class="card my-4">
+      <%= render partial: 'bio', locals: { user: @user } %>
+    </div>
+      <%= link_to "Add a new post", new_user_post_path(user_id: @user.id), class: "btn btn-primary"%>
+    <% if @posts.any? %>
+      <div class="post-list">
+        <% @user.recentposts.each do |post| %>
+          <%= render partial: 'post', locals: { user: @user, post: post } %>
+        <% end %>
+        <div class="text-center">
+            <%= link_to "See all posts", user_posts_path(@user), class: "btn btn-success" %>
+        </div>
+      </div>
+    <% else %>
+      <h2><%= @user.name %> has no posts</h2>
+    <% end %>
+  </div>
```

Comment:
- [ ] (Optional) Please make sure your User show page uses the method in the User model to return only the users 3 most recent posts according to requirement. Currently, all users posts are being shown


Your design
![image](https://github.com/Izebiafe/solo-blog/assets/81932478/d7daa516-ea38-4e74-9aba-ed970d2ea990)

Given frame
![image](https://github.com/Izebiafe/solo-blog/assets/81932478/fa4d7a84-42ea-4ed4-a7e5-4294f45d92ce)

ScreenCast
[cinnamon-2024-01-27T163401+0200.webm](https://github.com/Izebiafe/solo-blog/assets/81932478/6ac423e1-27e3-486d-bfe4-3ad59b88667e)